### PR TITLE
enable simplified-SDC step rejection of species increase

### PR DESCRIPTION
**Pull request by zingale**

### CODE REVIEW

**Comment by zingale** [comment]
tests: http://groot.astro.sunysb.edu/Microphysics/test-suite/gfortran/2021-08-05-003/index.html

**Comment by zingale** [comment]
Castro tests: http://groot.astro.sunysb.edu/Castro/test-suite/gfortran/2021-08-05-003/massive_star-sdc.html

File: integration/VODE/vode_dvstep.H [hunk 1]
```diff
@@ -254,11 +254,11 @@ int dvstep (burn_t& state, dvode_t& vstate)
 
             // this doesn't seem to work well with SDC
```
Comment (maxpkatz) [inline integration/VODE/vode_dvstep.H:255]: This comment would now be wrong.

**Comment by zingale** [comment]
I am also testing using looser criteria for SDC than Strang.

**Comment by zingale** [comment]
I think the proper way to do this is in #815

**Pull request closed without merging**